security-manager/src/main/java/org/wildfly/extension/security/manager/SecurityManagerExtension.java

```diff
@@ -64,6 +65,9 @@ public void initialize(final ExtensionContext context) {
         final ManagementResourceRegistration registration = subsystem.registerSubsystemModel(SecurityManagerRootDefinition.INSTANCE);
         registration.registerOperationHandler(GenericSubsystemDescribeHandler.DEFINITION, GenericSubsystemDescribeHandler.INSTANCE, false);
         subsystem.registerXMLElementWriter(SecurityManagerSubsystemParser.INSTANCE);
+
+        //Since version 1.0.0 of this subsystem has not been part of any product release I am not registering
+        //any transformers for it
```

Comment:
For the record: I discussed this with Jason and he agreed this is fine. We can look into transforming to WildFly 9.x if user demand arises.

I don't foresee ever transforming to 8.x.